src/scripts.js

```diff
@@ -1,12 +1,87 @@
 import {customersData, roomsData, bookingsData, userData, postData} from './api-calls';
 import './css/base.scss';
-import Hotel from '../classes/Hotel';
+import Hotel from './classes/Hotel';
+import domUpdates from './domUpdates';
+import image from './images/hotel-room.png';
+
+// Query Selectors:
+const homeView = document.querySelector('#homeView');
+const availableRoomsView = document.querySelector('#availableRoomsView');
+const title = document.querySelector('#title');
+const roomTypeTags = document.querySelector('#filterRoom');
+const calendar = document.querySelector('#dateCalander');
+
+
+// Buttons || icons:
+const homeBtn = document.querySelector('.home-btn');
+const myProfileBtn = document.getElementById('myProfileBtn');
+const signOutBtn = document.getElementById('signOutBtn');
+const submitRequestBtn = document.querySelector('#submitRequestBtn');
 
 let hotel; 
+let calendarDate;
 
 const getAllData = (userID) => {
-  Promise.all([customersData, roomsData, bookingsData, userData(userID)])
+  Promise.all([customersData(), roomsData(), bookingsData(), userData(userID)])
   .then(data => {
-    hotel = new Hotel(data[0], data[2], data[1])
+    hotel = new Hotel(data[0].customers, data[2].bookings, data[1].rooms)
+    hotel.getAllUserBookings(data[3])
+    hotel.getTotalPrice()
+    loadCustomerInfo()
+    // console.log(hotel.currentCustomer.bookings);
   })
-}
\ No newline at end of file
+}
+
+// Event Listeners:
+
+
+
+// Functions: 
+
+const bookRoom = event => {
```

Comment:
This is an awesome function. I like how you used the getAllData function to generate the updated data on the page